README.md

```diff
@@ -42,6 +42,18 @@ Build instructions and more details can be found in the
 * Step 1: Make a space flight simulator
 * Step 2: Bloat it with features
 
+## Compiling for Windows on Visual Studio
+You will need: cmake, Visual Studio
+* Clone the repo with `https://github.com/TheOpenSpaceProgram/osp-magnum`
+* Open the repo `cd osp-magnum`
+* Get submodules `git submodule init && git submodule update`
+* Create the build folder `mkdir build && cd build`
+* Download [SDL2 visual studio development binaries](https://www.libsdl.org/release/SDL2-devel-2.0.12-VC.zip), and extract it into the build folder
+* Run the command `cmake -DCMAKE_PREFIX_PATH=SDL2-2.0.12 -DSDL2_LIBRARY_RELEASE=SDL2-2.0.12/lib/x64/*.lib -DSDL2_INCLUDE_DIR=SDL2-2.0.12/include ..` to create project files
+* Open the solution `OSP-MAGNUM.sln` in file explorer, and visual studio should open up
+* Shift-control-B to build for debug. 
+* Copying files TODO
```

Comment:
@EhWhoAmI Maybe you'd like to add some logic to cmake to make it automatically copy the files in question, instead of the developer having to do it manually?

Reply:
There ought to be a better way, it's basically a quirk of how VS decides to structure its project data. The binaries from the 3rd party libraries have to be copied to the Build/ directory where VS puts `osp-magnum.exe`, we just have to figure out how to get VS to use the right directories; I can try to figure it out this weekend. On linux it all just works out of the box.

Reply:
I found the solution. I'm not sure why VS treats the projects differently, but osp-magnum doesn't follow the normal CMake build paths.

By default, the variable `CMAKE_RUNTIME_OUTPUT_DIRECTORY` is set to `<osp-magnum>/build/$<CONFIG>/bin`, where `$<CONFIG>` is whatever build configuration you're using in VS (e.g. Debug, Release). If you change the path to `/build/bin/$<CONFIG>` the binaries will all be built in the right place.

Before it would produce a directory structure like
```
build/
|--  bin
|     |-- Debug
|     |     +-  osp-magnum.exe
|     |-- Release
|           +- osp-magnum.exe
|-- Debug
|    |-- bin
|    |      +- <binaries>
|    |-- lib
|           +- <libraries>
|-- Release
     |-- <same as Debug>
```
so by switching the order of `$<CONFIG>/bin` to `bin/$<CONFIG>` it matches the osp-magnum project path and everything goes into the right folder

Reply:
It's also possible to just set the working directory to `osp-magnum/bin`

Reply:
That's necessary too. I forget how it works on linux since I haven't build the repo there in a while but on windows you need this separate build/ directory and VS creates a bunch of folders for the various build configurations. The issue is that for some reason it separates the main project (osp-magnum) from the rest of the projects in the solution, this change just fixes that issue but you still need to change the working directory to osp-magnum/bin. I think on linux I just run `../build/bin/osp-magnum.exe` from that directory or something.